src/site/content/en/handbook/inclusion-and-accessibility/index.md

```diff
@@ -1,41 +1,75 @@
-* For pieces up to 1,600 words, use [TextEvaluator](http://textevaluator.ets.org/TextEvaluator/).
-* For pieces up to 1,000 words, use [Lexile](https://lexile.com/educators/tools-to-support-reading-at-school/tools-to-determine-a-books-complexity/the-lexile-analyzer/).
+Use preferred words from [our word
+list](https://developers.google.com/style/word-list). This list is extensive
+and provides both explanations and alternatives.
+
+### Avoid easy, simple, and fast
+
+Don't descibe tasks as easy, simple, or fast. These are often subjective bias
+based on the writer's knoweldge and resources. It's possible the reader will
+see a description of a task as "easy" and feel badly if they don't find it
+easy to complete. 
+
+You can descibe certain things as "less complex" when compared to other
+versions. But, for the most part, if you believe something is "simple," the
+example should speak for itself instead of needing extra validation.
 
-It can be difficult to write technical content for a grade 8 reading level. The following guidelines help keep your language as accessible as it can be.
+Refer to [James Fisher's blog on Don't use simply](https://jameshfisher.com/2017/02/22/dont-use-simply/)
 
-Write shorter, simpler sentences.
+## Write readable language
+
+### Aim for a grade 8 reading level
+
+Aim for a grade 8 reading level when possible. If you don't have a feel for
+what grade 8 texts look like, check out some [examples on
+Newsela](https://newsela.com/articles/#/rule/latest?grade_levels=8.0). You can
+also run your text through a readability test.
+*  For pieces up to 1,600 words, use [TextEvaluator](http://textevaluator.ets.org/TextEvaluator/).
+*  For pieces up to 1,000 words, use [Lexile](https://lexile.com/educators/tools-to-support-reading-at-school/tools-to-determine-a-books-complexity/the-lexile-analyzer/).
+
+It can be difficult to write technical content for a grade 8 reading level.
+The following guidelines help keep your language as accessible as it can be.
+
+Write short and clear sentences.
 
 {% Compare 'worse', 'Don\'t' %}
 > We also include an HTML table of buttons for accessibility reasons that is on top of the canvases but is made invisible using opacity: 0.
 
 {% endCompare %}
 
 {% Compare 'better', 'Do' %}
-> We also include an HTML table of buttons to improve accessibility. The table sits on top of the canvases and is made invisible using opacity: 0.
+> Also, we include an HTML table of buttons to improve accessibility. The table sits on top of the canvases and is made invisible using opacity: 0.
 
 {% endCompare %}
 
-Use commonly used words that your [audience](/handbook/audience) is likely to be familiar with instead of specialized vocabulary. If you need help finding a more accessible alternative, the [Merriam-Webster Learner's Dictionary](http://learnersdictionary.com/) is a handy tool.
+### Favor known terms and avoid jargon
```

Comment:
Can you add a suggestion to avoid idioms as well?

Reply:
I see now that this is covered under inclusivity. I think it belongs in either place. Happy to keep it under inclusivity because it gets more attention that way.

Reply:
I agree it works in either place, and I hope it gets well-worthy attention in inclusivity. And that folks read this doc, at all!